content/posts/2022-10-08-why-good-code-style-matters.md

```diff
@@ -1,8 +1,11 @@
 ---
 title: Why good code style matters
-date: 2022-10-24T10:10:19.885Z
+date: 2023-10-24T10:10:19.885Z
+
 featuredImage: /post-images/good-style-article-prettier.png
-draft: false
+
+draft: true
+# !This code is for testing only DO NOT MERGE IT!
```

Comment:
Cancel these chances, and then you can merge this branch by yourself

You have my approval